backend/FileSystem/Path.php

```diff
@@ -4,26 +4,26 @@
 
 namespace Internships\FileSystem;
 
-define("PATHS_FOLDER_SEPARATOR", "/");
-
 class Path
 {
+    public const FOLDER_SEPARATOR = "/";
+
     public function __construct(
         protected string $root
     ) {
     }
 
     public function getFull($relativePath, bool $isFile = false): string
     {
-        $fullPath = $this->root . PATHS_FOLDER_SEPARATOR . $relativePath;
+        $fullPath = $this->root . self::FOLDER_SEPARATOR . $relativePath;
```

Comment:
Why not `static`?

Reply:
Ah, I didn't consider this file much when making changes. Static makes more sense if somebody would like different path separator for various reasons.